ios/Video/RCTVideo.swift

```diff
@@ -795,7 +802,11 @@ class RCTVideo: UIView, RCTVideoPlayerViewControllerDelegate, RCTPlayerObserverH
 
     func createPlayerViewController(player:AVPlayer, withPlayerItem playerItem:AVPlayerItem) -> RCTVideoPlayerViewController {
         let viewController = RCTVideoPlayerViewController()
+        #if os(tvOS)
+        viewController.showsPlaybackControls = true
+        #else
         viewController.showsPlaybackControls = false
+        #endif
```

Comment:
Another trick that you can use here, is to have a small variable or function e.g. `shouldUseCustomPlayer` and then inside of that function return `true` if `os(tvOS)` otherwise `false`. That can help readability + make the code more flexible.

```swift
viewController.showsPlaybackControls = shouldUseCustomPlayer
```